src/terraform/mlz/variables.tf

```diff
@@ -59,6 +53,14 @@ variable "create_assignment" {
   default     = false
 }
 
+variable "tags" {
+ description = "A map of key value pairs to apply as tags to resources provisioned in this deployment"
+ type        = map(string)
+ default = {
+  "DeploymentName" : "mlz-dev",
```

Comment:
What does deployment name mean, or what is it used for? I think the acceptance criteria are met without this tag but if it's a useful tag then I'm all for it. :)

Reply:
It is a pre-existing artifact. I don't think it is used for anything, will take `deploymentname` out and test and post new changes momentarily.

Reply:
I think the tag was previously used to track the value set with the `-z` option of `deploy.sh` (which currently is still used to tag some of the pre-Terraform resources like the RGs for config and TF state, and used by the `clean.sh` script to remove those resources.) But I think it's going to end up being not required for the resources deployed by Terraform.  

Thanks for testing this! I think removing it should work.

Reply:
It appears to be used to some extent in 
src/scripts/terraform/create_tfvars_from_config.sh:51
`append_kvp "deploymentname" "${mlz_env_name}"`
But as stated looks like `deploymentname` actually maps to a different value `mlz_env_name`
It's current use as a tag is ineffectual but in the midst of confirming now.